### PR TITLE
Marketplace vm solution v0.6.0

### LINKED ISSUE #349: Marketplace VM solution - 0.6.0
Release 6.0 has the following requirements:
1. Do all add-iam-policy-binding within the VM startup. I'm also wondering if that service account really needs the iam.securityAdmin vs just iam.ServiceAccountAdmin role

~~2. If you look at the Anthos solution the user can actually create the IAM service account via the marketplace UI. See the anthos-trial-on-gcp.jinja.schema.~~

3. Instead of making the GCE Service Account a user of the custom service account, just create the VM using the custom service account to run as the required account permissions. 

Only the first and third items are required, but the second item is helpful to simplify the prerequisites for the customer.

Additional Details for each item above:

1. The Cloud Build code that deploys the API creates a new service account and assigns permissions. This code should be moved to the VM startup script.
2. See this link on how to create the SA in the UI.
3. Compute instance should run as the SA when all commands are executed. This will remove the prerequisite step to include the default SAs (DM, Compute, Cloud Build) as service account user on the custom SA.  Deployment Manager, Cloud Build cannot use alternative SAs.

### PR DESCRIPTION
**Pull request by swilliams11**
### All Submissions:

* [X] Have you followed the guidelines in our Contributing document?
* [X] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### New Feature Submissions:

1. [X] Does your submission pass tests?
2. [X] Have you lint your code locally prior to submission?

Resolves #349

### CODE REVIEW

**Comment by swilliams11** [comment]
Fixes #349

File: marketplace/releases/RELEASE_NOTES_VM_SOLUTION.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
 # Google Cloud Platform Release Notes for Marketplace VM Solution
 
+## 0.6.0 Release Notes
+* Build - 10/26/2020
+
+### Datashare
+* Datashare release version is 0.5.0
```
Comment (mservidio) [inline marketplace/releases/RELEASE_NOTES_VM_SOLUTION.md:7]: Let's change to 0.5.1

**Review by mservidio — CHANGES_REQUESTED** [review]
Let's update the Datashare version to [0.5.1](https://github.com/GoogleCloudPlatform/datashare-toolkit/releases/tag/v0.5.1).

**Comment by swilliams11** [comment]
@mservidio Actually, its too late to change this.  Marketplace release 0.6.0 is already published to the Marketplace with v0.5.0, so I have to publish a new Marketplace release (v0.6.1) and have that point to the Datashare release version 0.5.1.

**Review by mservidio — APPROVED** [review]
LGTM

**Pull request merged**